docs/nodes/db-neo4j-usage.md

```
@@ -0,0 +1,83 @@
+  WHEN 'CONCENTRATION_RISK' THEN 1
+  ELSE 99
+END
+RETURN count(c) AS updated
+```
+
+## Two db_neo4j Nodes in One Pipeline
+
+Neo4j appears twice in the Portfolio Brain cascade pipeline — once to read patterns,
+once to write results. This is a common pattern:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)